rclpy/docs/source/conf.py

```diff
@@ -194,9 +194,9 @@
 
 # -- Extension configuration -------------------------------------------------
 
-autoclass_content = 'both'
+#autoclass_content = 'both'
 
 autodoc_default_options = {
-    'members': None,
-    'undoc-members': True,
+    'members': True,  # document members
```

Comment:
I've tried reverting this change and the `qos_profile_*` variables still appear in the docs. I think it is the addition of the `#:` comments that is making them appear, and this line has no affect on them. I think we can undo this change.

Reply:
>  I think it is the addition of the `#:` comments that is making them appear, and this line has no affect on them.

True, still: `None` and `True` keywords have the same meaning. I would stick to one of them to avoid confusion. What do you think?

Reply:
I think it makes more sense to include this minor change in another commit.

Reply:
> I think it makes more sense to include this minor change in another commit.

Sounds good to me.